python/sglang/srt/layers/quantization/fp8.py

```diff
@@ -217,7 +217,7 @@ def create_weights(
 
         # WEIGHT
         weight_dtype = (
-            torch.float8_e4m3fn
+            torch.float8_e4m3fnuz if is_hip() else torch.float8_e4m3fn
```

Comment:
We should not have this, serialized weight is always OCP (torch.float8_e4m3fn)

Reply:
it would encounter the error "python/sglang/srt/layers/quantization/fp8_kernel.py:176:33: error: Unsupported conversion from 'f8E4M3FN' to 'f16'
        accumulator += tl.dot(a, b) * a_s[:, None] * b_s[None, :]" with ```torch.float8_e4m3fn``` at [w8a8_block_fp8_matmul](https://github.com/sgl-project/sglang/blob/main/python/sglang/srt/layers/quantization/fp8_utils.py#L50)

Reply:
Please check how `normalize_e4m3fn_to_e4m3fnuz` is used.
Basically - we do not expected non-OCP/e4m3fn dtype in the quantized model.